static/src/javascripts-legacy/projects/commercial/modules/high-merch.js

```diff
@@ -67,7 +67,9 @@ define([
             if (args) {
                 fastdom.write(function () {
                     args[1].parentNode.insertBefore(args[0], args[1].nextSibling);
-                    addSlot(args[0]);
+                })
+                .then(function () {
+                    addSlot(args[0], true);
```

Comment:
1- Yes 😄 
2- I'm taking the habit of moving everything that is not touching the DOM out `fastdom` frames.